Formula/c/calceph.rb

```diff
@@ -11,30 +11,27 @@ class Calceph < Formula
   end
 
   bottle do
-    sha256 cellar: :any,                 arm64_sonoma:   "6caee108ddd6a6510998c572929ffd950f76040c267788098b8a117ec0197fc6"
-    sha256 cellar: :any,                 arm64_ventura:  "f003f9a8037eace25bc98d659a3afb8776669278da4f1928a5976965c8035ab3"
-    sha256 cellar: :any,                 arm64_monterey: "cd5dbd77c1e6989284f95239c1327cdef01bedb1706bc756544518b003a3f27e"
-    sha256 cellar: :any,                 sonoma:         "f55f8ccf745f4815d92a193b699d296f7c0a14b860048f482b50b9454739f738"
-    sha256 cellar: :any,                 ventura:        "cf5a9ff7639306750dc9996946ab150dee3d08c967a64e463c25a3959ee334bf"
-    sha256 cellar: :any,                 monterey:       "080c50596ddb83e1741018175e8219419f57bddfc2a72201d7285b5a1b58d0a4"
-    sha256 cellar: :any_skip_relocation, x86_64_linux:   "6d889b6e08767bd3c7353cfc85d075e5c6bb181ca33220b64809cae1c303be09"
+    sha256 cellar: :any,                 arm64_sonoma:   "23612ab3cdac456c9fc5fd34554bc8dc4523c9b4ee7b0afa39595fbff09da603"
+    sha256 cellar: :any,                 arm64_ventura:  "9ed7d5b25a2b4c07f614ec427d828104722030db63c3cd2437c7c015f70c7b8e"
+    sha256 cellar: :any,                 arm64_monterey: "feaa4b155cddb90b7e2fd40d73608d081fc007ff89aefd3fb1f90bc4a3134cc4"
+    sha256 cellar: :any,                 sonoma:         "aff5beada38dbfd166dd3417bb17a32e0fddd2fce41028e78fd472dc833d22b3"
+    sha256 cellar: :any,                 ventura:        "7435bf4fd20c10e3bafb058f1086999afae1d0c67679d628d8ae0f213a03bf1c"
+    sha256 cellar: :any,                 monterey:       "e2d5efc2ae19879722204a31044870d989ccf91d8a95edfe9bd500754f1c6444"
+    sha256 cellar: :any_skip_relocation, x86_64_linux:   "591da9c8bc2ef9191bcf0e186115a50424a819aef4201625a75a0d514b494f78"
   end
 
+  depends_on "cmake" => :build
   depends_on "gcc" # for gfortran
 
-  # Fix -flat_namespace being used on Big Sur and later.
-  patch do
-    url "https://raw.githubusercontent.com/Homebrew/formula-patches/03cf8088210822aa2c1ab544ed58ea04c897d9c4/libtool/configure-pre-0.4.2.418-big_sur.diff"
-    sha256 "83af02f2aa2b746bb7225872cab29a253264be49db0ecebb12f841562d9a2923"
-  end
-
   def install
-    system "./configure", "--disable-debug",
-                          "--disable-dependency-tracking",
-                          "--disable-silent-rules",
-                          "--prefix=#{prefix}"
-    system "make"
-    system "make", "install"
+    args = %w[
+      -DBUILD_SHARED_LIBS=ON
+      -DENABLE_FORTRAN=ON
+      -DCMAKE_INSTALL_RPATH=#{rpath}
+    ]
```

Comment:
This should be a `%W` array instead of `%w`. #169953

Reply:
make sense 👍